assets/bloomberg.yaml

```diff
@@ -0,0 +1,131 @@
+---
+
+- type: dataset
+  name: FinPile
+  # General
+  organization: Bloomberg
+  description: A comprehensive dataset consisting of a range of English financial
+    documents including news, filings, press releases, web-scraped financial documents,
+    and social media drawn from the Bloomberg archives that was used to train the
+    BloombergGPT model.
+  created_date:
+    value: 2023-03-30
+    explanation: The date BloombergGPT was announced in the [[Bloomberg article]](https://www.bloomberg.com/company/press/bloomberggpt-50-billion-parameter-llm-tuned-finance/).
+  url: https://arxiv.org/pdf/2303.17564.pdf#section.2
+  datasheet:
+    value: https://arxiv.org/pdf/2303.17564.pdf#section.2
+    explanation: Section 2 of the BloombergGPT paper.
+  modality: text (English)
+  size:
+    value: 363B tokens
+    explanation: See [[Financial Datasets]](https://arxiv.org/pdf/2303.17564.pdf#subsection.2.1)
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: []
+  license: unknown
+  included:
+    value: |
```

Comment:
Just curious: what does the `|` syntax do compared to `>`?

Reply:
I used it to create bulleted/numbered lists like this:
![image](https://user-images.githubusercontent.com/51331134/230704481-9db745c6-7ad3-49b9-a086-e27ff473b47b.png)

`|` is supposed to preserve all newline characters, unlike `>` which converts newlines to spaces, but it only seemed to work for lists here unsure why.

Reply:
Ah got it, good to know!